src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -433,6 +433,9 @@ protected net.minecraft.world.level.levelgen.NoiseSampler f_158643_ # noiseSetti
 public net.minecraft.world.level.levelgen.NoiseSampler f_158644_ # blendedNoise
 public net.minecraft.world.level.levelgen.NoiseSampler f_158645_ # islandNoise
 protected net.minecraft.world.level.levelgen.NoiseSampler m_158667_(DI)D # applySlide
+public net.minecraft.world.level.levelgen.WorldGenSettings f_64606_ # legacyCustomOptions
+public net.minecraft.world.level.levelgen.WorldGenSettings <init>(JZZLnet/minecraft/core/MappedRegistry;Ljava/util/Optional;)V # constructor
+public net.minecraft.world.level.levelgen.feature.blockplacers.BlockPlacerType <init>(Lcom/mojang/serialization/Codec;)V # constructor
```

Comment:
Was this added as a comment from someone on discord? Otherwise, it's a bit out of scope for this PR.

Reply:
Think that was a rebasing flub, I'll clean that up